src/test/tests/databases/mfem.py

```diff
@@ -262,8 +265,53 @@ def test_mfem_lor_field(tag_name, dbfile):
     readOptions["MFEM LOR Setting"] = "MFEM LOR"
     SetDefaultFileOpenOptions("MFEM", readOptions)
 
+
+
 TestSection("Legacy and New LOR Fields")
 for dbfile in ex01_results:
     test_mfem_lor_field("LOR_Fields", dbfile)
 
+
+def test_mfem_quad_func(tag_name, dbfile, var_name):
+    ResetView()
+    base = os.path.splitext(os.path.basename(dbfile))[0]
+    OpenDatabase(dbfile)
+    # get the mesh metadata and find the mesh with "quad_func"
+    md = GetMetaData(dbfile)
+    plot_type = ""
+    qf_mesh_name = ""
+    for i in range(md.GetNumScalars()):
+        smd = md.GetScalars(i)
+        if smd.name == var_name:
+            plot_type = "Pseudocolor"
+            qf_mesh_name = smd.meshName
+    for i in range(md.GetNumVectors()):
+        vmd = md.GetVectors(i)
+        if vmd.name == var_name:
+            plot_type = "Vector"
+            qf_mesh_name = vmd.meshName
```

Comment:
I have the same questions about these loops as I do for the blueprint tests.

Reply:
The current datasets only have one field, it is either a scalar field or a vector field. 
The goal is to figure out which case, I think this code is safe with the current example data.

Reply:
Could you add a comment to that effect? It could be confusing down the road potentially unless you know that specific info.

Reply:
should we add the same comment to these loops also?